app/models/katello/content_view_erratum_filter.rb

```diff
@@ -30,13 +30,18 @@ def content_unit_pulp_ids(repo)
       if filter_by_id?
         errata_ids = erratum_rules.map(&:errata_id)
         errata_pulp_ids = errata_package_pulp_ids_from_errata_ids(repo, errata_ids)
+        errata_pulp_ids += errata_module_stream_pulp_ids_from_errata_ids(errata_ids)
       else
         clauses = []
         clauses << errata_from
         clauses << errata_to
         clauses << types_clause
         package_filenames = Erratum.list_filenames_by_clauses(repo, clauses.compact)
         errata_pulp_ids = errata_package_pulp_ids_from_package_filenames(repo, package_filenames)
+        errata_pulp_ids += errata_module_stream_pulp_ids_from_clauses(repo, clauses.compact)
+      end
+      100.times do
```

Comment:
What is this `100.times` doing?

Reply:
Aah..Was doing some caveman debugging..Removing it..:)